src/controllers/commands/builtin/twitch/Announce.cpp

```diff
@@ -11,7 +11,8 @@
 
 namespace chatterino::commands {
 
-QString sendAnnouncement(const CommandContext &ctx)
+QString sendAnnouncementColor(const CommandContext &ctx,
```

Comment:
nit: move this function to an anon namespace, outsode if `namespace chatterino::commands`

so it'd be something along the lines of

```
namespace {
using namespace chatterino;
using namespace chatterino::commands; // maybe

QString sendAnnouncementColor...
```